src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter/Main.cs

```diff
@@ -56,15 +57,32 @@ public List<Result> Query(Query query)
                 .ToList();
         }
 
+        private string GetOunceUnitNameString(UnitConversionResult result)
```

Comment:
Please name this more general to allow using it later for other units too.

Reply:
Named it `GetUnitNameString`